parts/inc/call

```diff
@@ -405,6 +426,32 @@ if ($] ge '5.007003' or ($] ge '5.006001' and $] lt '5.007')) {
     skip 'skip: no support for references in $@', 0 for 1..7;
 }
 
+ok(eval { Devel::PPPort::eval_sv('die', 0); 1 });
+ok(!eval { Devel::PPPort::eval_sv('die', &Devel::PPPort::G_RETHROW); 1 });
+ok($@ =~ /^Died at \(eval [0-9]+\) line 1\.\n$/);
+ok(eval { $@ = 'string1'; Devel::PPPort::eval_sv('', 0); 1 });
+ok(eval { $@ = 'string1'; Devel::PPPort::eval_sv('', &Devel::PPPort::G_RETHROW); 1 });
+ok(eval { $@ = 'string1'; Devel::PPPort::eval_sv('$@ = "string2"', 0); 1 });
+ok(eval { $@ = 'string1'; Devel::PPPort::eval_sv('$@ = "string2"', &Devel::PPPort::G_RETHROW); 1 });
+ok(eval { $@ = 'string1'; Devel::PPPort::eval_sv('$@ = "string2"; die "string3"', 0); 1 });
+ok(!eval { $@ = 'string1'; Devel::PPPort::eval_sv('$@ = "string2"; die "string3"', &Devel::PPPort::G_RETHROW); 1 });
+ok($@ =~ /^string3 at \(eval [0-9]+\) line 1\.\n$/);
+
+if ($] ge '5.007003' or ($] ge '5.006001' and $] lt '5.007')) {
```

Comment:
The thing that works in most releases is '$]' >= 5.007003

Reply:
This code branch is working fine also in 5.6.2, see travis tests.
So there no need to limit minimal version to 5.7.3.

Reply:
I'm sorry I wasn't clear about the point.  You used the string comparison operators; it's better to use the numeric ones,, so NOT 'ge' but '>=''.  I had no comment on the operand values

But shortly there will be an ivers() available to .t files, so you can say ivers($]) >= ivers(5.7.3), for example and things will work.

Reply:
Thanks! Understood now.

Reply:
Now I fixed it and updated PR.

Reply:
But actually, my example is wrong because on some versions of perl, that ivers(5.7.3) is turned into a vstring.  And the problem is the behavior varies depending on perl version.  So something that could be a vstring has to be in quotes.  So my example should should have been

ivers($]) >= ivsers("5.7.3")

Reply:
Zefram has said in the past that the best way to do version comparisons is like so: `"$]" >= 5.007003` -- that is, stringify the version first, and then perform a numeric comparison.